cmd/tetra/tracingpolicy/tracingpolicy.go

```diff
@@ -97,6 +108,15 @@ func addTracingPolicy(ctx context.Context, client tetragon.FineGuidanceSensorsCl
 	}
 }
 
+func deleteTracingPolicy(ctx context.Context, client tetragon.FineGuidanceSensorsClient, name string) {
+	_, err := client.DeleteTracingPolicy(ctx, &tetragon.DeleteTracingPolicyRequest{
+		Name: name,
+	})
+	if err != nil {
+		fmt.Printf("failed to delete tracing policy: %s\n", err)
```

Comment:
A nit, sorry for the back and forth I just noticed that, a non-blocker because we do the same for `addTracingPolicy`. 

This is out of scope of this PR because the whole cmd need rework but: we should use `common.CliRunErr` in the `cobra.Command` and return an error to the `Run` function so that the command returns a non-0 return code on errors.

The most simple way of getting a non-0 return code here would maybe to use:
```go
logger.GetLogger().WithFields(logrus.Fields{"name": name }).WithError(err).Fatal("failed to delete tracing policy")
```

Reply:
Yeah, I think `Fatal` is the right thing to do if we want an error code. I'd say, let's merge the PR as it is and we can deal with above elsewhere.

Reply:
yep because overall the whole thing need a bit of rework for other functions that do return 0 on success.

Reply:
created https://github.com/cilium/tetragon/issues/1262